### PR TITLE
feat(oxc_semantic): Improve sample visualization

### PR DESCRIPTION
**Pull request by TzviPM**
1. add a `test.js` file to the project root:

```js
class A extends B {
  constructor() {
    try {
      super();
    } finally {
      this.a;
    }
  }
}
```

2. run:

```bash
$ cargo run -p oxc_semantic --example simple
   Compiling oxc_semantic v0.5.0 (/home/tzvipm/src/github.com/tzvipm/oxc/crates/oxc_semantic)
    Finished dev [unoptimized + debuginfo] target(s) in 32.07s
     Running `target/debug/examples/simple`
Wrote AST to: test.ast.txt
Wrote CFG blocks to: test.cfg.txt
Wrote CFG dot diagram to: test.dot
```

3. resulting graph from .dot file:

image

### CODE REVIEW

File: crates/oxc_semantic/examples/simple.rs
Comment (Boshen) [inline crates/oxc_semantic/examples/simple.rs:1]: This isn't simple anymore, shall we add a cfg.rs?

Comment (TzviPM) [reply crates/oxc_semantic/examples/simple.rs:1]: Ah, sure!

Comment (TzviPM) [reply crates/oxc_semantic/examples/simple.rs:1]: Done :)

File: crates/oxc_semantic/examples/simple.rs [hunk 1]
```diff
@@ -21,7 +21,6 @@ fn main() {
     let ret = Parser::new(&allocator, &source_text, source_type).parse();
 
     let program = allocator.alloc(ret.program);
-    println!("{:#?}", &program);
```
Comment (TzviPM) [inline crates/oxc_semantic/examples/simple.rs:24]: I committed this by mistake in a previous PR

**Review by Boshen — APPROVED** [review]

**Pull request merged**